burn-compute/src/tune/tune_cache.rs

```diff
@@ -1,13 +1,36 @@
+#[cfg(feature = "autotune-persistent-cache")]
+mod std_imports {
+    pub use std::fs;
+    pub use std::fs::File;
+    pub use std::io;
+    pub use std::path::Path;
+    pub use std::path::PathBuf;
+}
+
+#[cfg(feature = "autotune-persistent-cache")]
+use std_imports::*;
+
 use super::AutotuneKey;
 use super::AutotuneOperation;
 use super::AutotuneOperationSet;
 use alloc::boxed::Box;
 use hashbrown::HashMap;
 
+/// Return the file path for the persistent cache on disk
+#[cfg(feature = "autotune-persistent-cache")]
+pub fn get_persistent_cache_file_path() -> PathBuf {
+    let home_dir = dirs::home_dir().expect("Could not get home directory");
+    let path_dir = home_dir.join(".cache").join("burn").join("autotune");
+    let path = Path::new(&path_dir);
+    path.join("autotune-cache.json")
+}
+
 /// Use to find and reuse the best kernel for some input
-#[derive(Debug, Default)]
+#[derive(Debug)]
 pub(crate) struct TuneCache<K> {
-    cache: HashMap<K, usize>,
+    cache: HashMap<K, (bool, usize)>,
```

Comment:
Maybe make a struct CacheResult that has is_checked and index fields? It would be clearer than a tuple

Reply:
Also maybe give cache a name that distinguishes it better from persistent_cache, like in_memory_cache

Reply:
Done, that's way better 👍🏻